app/models/volunteer.rb

```diff
@@ -15,7 +15,7 @@ class Volunteer < ApplicationRecord
   AVAILABILITY = [:flexible, :morning, :afternoon, :evening, :workday, :weekend].freeze
   SALUTATIONS = [:mrs, :mr].freeze
 
-  enum acceptance: { undecided: 0, accepted: 1, rejected: 2, resigned: 3 }
+  enum acceptance: { undecided: 0, invited: 1, accepted: 2, rejected: 3, resigned: 4 }
```

Comment:
Note that existing records in the DB might have the wrong state if you change the numeric values.